src/GraphQL.Serialization.Newtonsoft/ExecutionResultJsonConverter.cs

```diff
@@ -99,11 +99,11 @@ private void WriteErrorExtensions(ExecutionError error, JsonWriter writer, JsonS
                 serializer.Serialize(writer, error.Code);
             }
 
-            if (error.HasCodes)
+            if (error.HasCodes())
```

Comment:
No need to change - just make these properties public in original class and then ignore them in the created contract resolver.

Reply:
This is reverted